Vagrantfile

```diff
@@ -31,7 +39,11 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     v.vm.box_url = "http://files.vagrantup.com/precise64_vmware.box"
   end
 
-  config.vm.synced_folder ".", "/srv", owner: 'www-data', group: 'www-data', mount_options: ["dmode=775", "fmode=664"]
+  if (/cygwin|mswin|mingw|bccwin|wince|emx/ =~ RUBY_PLATFORM) == nil
```

Comment:
Vagrant should actually handle this platform check for us -- it automatically falls back to virtualbox shares on unsupported platforms.